CHANGES.md

```diff
@@ -14,7 +14,25 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
-[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+  ###  Changes
+
+  - Notebook deployment: allow to specify required branch for any tutorial
+    notebook repos in `env.local`.
+
+    Examaple: set `WORKFLOW_TESTS_BRANCH` and any other
```

Comment:
typo

Reply:
It was intentional to test if you are paying attention or not :D